packages/cozy-stack-client/src/SettingsCollection.js

```diff
@@ -25,6 +41,33 @@ class SettingsCollection extends DocumentCollection {
       )
     }
   }
+
+  /**
+   * Updates a settings document
+   *
+   * @param {object} document - Document to update. Do not forget the _id attribute
+   */
+  async update(document) {
+    let resp
+
+    if (document._id === 'io.cozy.settings.instance') {
```

Comment:
I did not see other PUT which could be used like this in cozy stack documentation.